test/inductor/test_torchinductor_opinfo.py

```diff
@@ -379,15 +378,13 @@ def get_skips_and_xfails(from_dict, xfails=True):
 )
 
 
-def wrapper_set_seed(op, *args, **kwargs):
-    """Wrapper to set seed manually for some functions like dropout
-    See: https://github.com/pytorch/pytorch/pull/62315#issuecomment-896143189 for more details.
-    """
-    torch.manual_seed(42)
+def wrapper_noop_set_seed(op, *args, **kwargs):
     return op(*args, **kwargs)
 
 
-torch.testing._internal.common_methods_invocations.wrapper_set_seed = wrapper_set_seed
+torch.testing._internal.common_methods_invocations.wrapper_set_seed = (
```

Comment:
We should be patching this inside of the test, `with unittest.patch`, so we dont inadvertently affect other test  and maybe be calling `freeze_rng_state` as well